src/Components/DropdownField.res

```diff
@@ -24,13 +43,31 @@ let make = (
     let target = ev->ReactEvent.Form.target
     let value = target["value"]
     setValue(value)
+    if isDisplayValueVisible {
+      let findDisplayValue =
+        options
+        ->Array.find((ele: optionType) => ele.value === value)
+        ->Option.getOr(defaultValue)
+
+      switch setDisplayValue {
+      | Some(setDisplayValueFun) =>
```

Comment:
No need to add Fun at the end of the name. Either remove it or change it to Func

Reply:
Updated